jest.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.tsx?$',
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
   setupFiles: ['<rootDir>/src/setupTests.ts'],
+  setupFilesAfterEnv: ['./jest.setup.js'],
```

Comment:
Instead of creating a new setup file, we could move the contents of `jest.setup.js` to `src/setupTests.ts` (and use an `import` instead of a `require`). As jest-axe needs `expect` in the environment in order to add it's extensions, we could move the existing `setupFiles` to `setupFilesAfterEnv` (so it'll look like `setupFilesAfterEnv: ['<rootDir>/src/setupTests.ts']`).